org/mozilla/jss/provider/javax/crypto/JSSKeyManager.java

```diff
@@ -69,6 +69,16 @@ public String chooseServerAlias(String keyType, Principal[] issuers, Socket sock
         return null;  // not implemented
     }
 
+    public org.mozilla.jss.crypto.X509Certificate getCertificate(String alias) {
+        try {
+            CryptoManager cm = CryptoManager.getInstance();
+            return cm.findCertByNickname(alias);
+        } catch (Exception e) {
+            logger.error(e.getMessage(), e);
+            throw new RuntimeException(e);
```

Comment:
@edewata Is there a better exception here? The existing KeyManager code doesn't raise an exception, and this'll get called fairly early in the initialization (though, in Tomcat, only when the first inbound connection happens).

Thoughts?